### PR TITLE
Use angle vector duration when no time is set.

### PR DESCRIPTION
**Pull request by aginika**
Copy the function from jsk_calibration and use it.
Tested on baxter

### CODE REVIEW

**Comment by garaemon** [comment]
I will remove the function from jsk_calibration.

✉︎ Ryohei

**Comment by garaemon** [comment]
this method should be useful for everybody.It can compute duration based on distace in c-space and joint velocity limit


✉︎ Ryohei

**Comment by aginika** [comment]
I updated the code. as https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/86#issuecomment-69718776

**Comment by garaemon** [comment]
any update about this issue?

**Comment by aginika** [comment]
I updated. Use :fast option to use optimized time.

**Comment by garaemon** [comment]
Because of euslisp update, test fails

**Comment by k-okada** [comment]
please update .travis directory @aginika

--
◉ Kei Okada

**Comment by garaemon** [comment]
[This PR](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/90
) is a workaround for testing

**Comment by garaemon** [comment]
Please merge origin/master

**Comment by aginika** [comment]
merged and add below documentation.

```
 (:angle-vector
     ;;tm (time to goal in msec)                                                                                                                                              
     ;;if designated tm is faster than fastest speed, use fastest speed                                                                                                       
     ;;if not specified, it will use 1/5 of the fastest speed .                                                                                                               
     ;;if :fastest is specefied use fastest speed calcurated from max speed       
```

**Comment by garaemon** [comment]
great commit

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -190,19 +190,45 @@
...
-         (diff-av (v- av (or (send self :state :potentio-vector) (send robot :angle-vector)))))
-     (dolist (jt (send robot :joint-list))
-       (cond ((derivedp jt linear-joint) ;; msec = mm / [m/sec]
-              (setq tm (max tm (abs (/ (elt diff-av idx) (send jt :max-joint-velocity))))))
-             ((derivedp jt rotational-joint) ;; msec = deg2rad(deg) / [rad/s] / 0.001
-              (setq tm (max tm (abs (/ (deg2rad (elt diff-av idx))
-                                       (send jt :max-joint-velocity) 0.001))))))
-       (incf idx)
-       ))
+   ;;tm (time to goal in msec)
```
Comment (k-okada) [inline pr2eus/robot-interface.l]: comment shouldd be "...", see 
for example https://github.com/euslisp/jskeus/pull/138/files#diff-84175dc49eace6b214a353e6d2b5fb34R923 goes to http://euslisp.github.io/jskeus/jmanual-node170.html#52679

**Comment by garaemon** [comment]
@aginika it would be good if :angle-vector-sequence have same interface

**Comment by aginika** [comment]
Ok, I will repair 2 point in other pull request.

**Pull request merged**